src/index/types/json/markdown.ts

```diff
@@ -142,6 +142,10 @@ export interface JsonMarkdownListItem {
      * this is a root element of the list starting at line 7.
      */
     $parentLine: number;
+    /** the contents of this list item, with the list marker and indentation stripped. */
+    $text?: string;
+    /** the symbol of this list item */
+    $symbol?: string;
```

Comment:
What is symbol? Is that whatever is inside of the `[ ]`?

Reply:
`symbol` is whatever comes before the `[ ]`, i.e.:
```
* [ ]
- [ ]
+ [ ]
^ these bits here at the beginning
```